python/semantic_kernel/kernel.py

```diff
@@ -641,6 +641,21 @@ def create_function_from_prompt(
 
         return function
 
+    def create_function_from_prompt_yaml(self, text: str, **kwargs: Any) -> "KernelFunction":
```

Comment:
there should also be a part of this that loads from disk, similar to how the existing plugins are loaded.

Reply:
I think it could be its own method that loads yaml files, and not need to be added to the existing method that loads the config.json/skprompt.txt.

Reply:
Indeed, that is also what I meant, but we do need a loader function from disk, not just from text.